### PR TITLE
Bugfix user birth day has no limit

### PR DESCRIPTION
**Pull request by dybi**
resolves:
https://github.com/Code-Poets/sheetstorm/issues/225

### CODE REVIEW

**Comment by kbeker** [comment]
@dybi Also please remember when you are creating PR, please add milestone, projects, label and assign yourself to this PR. Same thing please do with issue which is related to this PR. :)

File: employees/tests/test_unit_report_serializer.py [hunk 1]
```diff
@@ -22,16 +22,9 @@
...
-        "date": datetime.datetime.now().date(),
-        "description": "Some description",
-        "author": None,
-        "project": None,
-        "work_hours": Decimal("8.00"),
-    }
 
     def setUp(self):
-
+        super().setUp()
```
Comment (rwrzesien) [inline employees/tests/test_unit_report_serializer.py]: :+1:

File: users/common/strings.py [hunk 2]
```diff
@@ -1,46 +1,44 @@
 # pylint: disable=line-too-long
-from django.utils.translation import ugettext_lazy
+from django.utils.translation import ugettext_lazy as _
```
Comment (rwrzesien) [inline users/common/strings.py]: :+1: :+1: :+1:

File: users/serializers.py [hunk 3]
```diff
@@ -43,6 +45,13 @@ def validate_email(self, email: str) -> str:
                 raise serializers.ValidationError(CustomValidationErrorText.VALIDATION_ERROR_SIGNUP_EMAIL_MESSAGE)
         return email
 
+    def validate_date_of_birth(self, date_of_birth: datetime.date) -> datetime.date:  # pylint: disable=no-self-use
```
Comment (rwrzesien) [inline users/serializers.py]: This should be done directly on `CustomUser.date_of_birth` field as `validators=[MaxValidator(), MinValidator]` (https://docs.djangoproject.com/en/2.2/ref/validators/#maxvaluevalidator). This way we control it on lowest possible level in the app and don't have to worry about it anywhere else. If for some reason it won't work there, then `clean()` method of the model. The case is that we want to remove all remaining of `django-rest-framework` so this serializer will be removed soon, and we shouldn't loose this logic.

**Comment by rwrzesien** [comment]
@dybi Looks good, please just move the check to model instead of serializer, as it will be removed soon.

File: users/serializers.py (hunk 3 above)
Comment (dybi) [reply users/serializers.py]: @rwrzesien  Ok, I will move it, but ther's a problem with `MinValueValidator / MaxValueValidator`. Namely, the min and max value are not set in stone, they depend on the current date (for example, if yesterday user was not 18 years old validation should rasie exception, but if today user turned 18, it should pass).

I will create a custom validator and use it in model.

**Pull request merged**